CRITICAL - 2023-03-08 05:52:55 --> A non-numeric value encountered
in APPPATH/Models/Search.php on line 416.
 1 APPPATH/Models/Search.php(416): CodeIgniter\Debug\Exceptions->errorHandler(2, 'A non-numeric value encountered', 'APPPATH/Models/Search.php', 416, [...])
 2 APPPATH/Models/Telegram.php(47): App\Models\Search->addPokazC('142', 'SELECT counter.id, counterId, counterName FROM counter JOIN pokaz ON counter.id = cId
WHERE unit = 131 AND typeC = 8 AND ts != \'2023-02-01\' GROUP BY counter.id', '2023', '02')
 3 APPPATH/Models/Telegram.php(13): App\Models\Telegram->message(Object(stdClass), Object(stdClass), 971483502)
 4 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(Object(stdClass))
 5 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 6 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
 7 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-08 05:55:17 --> A non-numeric value encountered
in APPPATH/Models/Search.php on line 416.
 1 APPPATH/Models/Search.php(416): CodeIgniter\Debug\Exceptions->errorHandler(2, 'A non-numeric value encountered', 'APPPATH/Models/Search.php', 416, [...])
 2 APPPATH/Models/Telegram.php(47): App\Models\Search->addPokazC('142', 'SELECT counter.id, counterId, counterName FROM counter JOIN pokaz ON counter.id = cId
WHERE unit = 131 AND typeC = 8 AND ts != \'2023-02-01\' GROUP BY counter.id', '2023', '02')
 3 APPPATH/Models/Telegram.php(13): App\Models\Telegram->message(Object(stdClass), Object(stdClass), 971483502)
 4 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(Object(stdClass))
 5 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 6 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
 7 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-08 05:57:27 --> A non-numeric value encountered
in APPPATH/Models/Search.php on line 416.
 1 APPPATH/Models/Search.php(416): CodeIgniter\Debug\Exceptions->errorHandler(2, 'A non-numeric value encountered', 'APPPATH/Models/Search.php', 416, [...])
 2 APPPATH/Models/Telegram.php(47): App\Models\Search->addPokazC('142', 'SELECT counter.id, counterId, counterName FROM counter JOIN pokaz ON counter.id = cId
WHERE unit = 131 AND typeC = 8 AND ts != \'2023-02-01\' GROUP BY counter.id', '2023', '02')
 3 APPPATH/Models/Telegram.php(13): App\Models\Telegram->message(Object(stdClass), Object(stdClass), 971483502)
 4 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(Object(stdClass))
 5 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 6 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
 7 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-08 05:59:34 --> A non-numeric value encountered
in APPPATH/Models/Search.php on line 416.
 1 APPPATH/Models/Search.php(416): CodeIgniter\Debug\Exceptions->errorHandler(2, 'A non-numeric value encountered', 'APPPATH/Models/Search.php', 416, [...])
 2 APPPATH/Models/Telegram.php(47): App\Models\Search->addPokazC('142', 'SELECT counter.id, counterId, counterName FROM counter JOIN pokaz ON counter.id = cId
WHERE unit = 131 AND typeC = 8 AND ts != \'2023-02-01\' GROUP BY counter.id', '2023', '02')
 3 APPPATH/Models/Telegram.php(13): App\Models\Telegram->message(Object(stdClass), Object(stdClass), 971483502)
 4 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(Object(stdClass))
 5 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 6 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
 7 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-08 06:04:24 --> A non-numeric value encountered
in APPPATH/Models/Search.php on line 416.
 1 APPPATH/Models/Search.php(416): CodeIgniter\Debug\Exceptions->errorHandler(2, 'A non-numeric value encountered', 'APPPATH/Models/Search.php', 416, [...])
 2 APPPATH/Models/Telegram.php(47): App\Models\Search->addPokazC('142', 'SELECT counter.id, counterId, counterName FROM counter JOIN pokaz ON counter.id = cId
WHERE unit = 131 AND typeC = 8 AND ts != \'2023-02-01\' GROUP BY counter.id', '2023', '02')
 3 APPPATH/Models/Telegram.php(13): App\Models\Telegram->message(Object(stdClass), Object(stdClass), 971483502)
 4 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(Object(stdClass))
 5 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 6 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
 7 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-08 06:06:55 --> A non-numeric value encountered
in APPPATH/Models/Search.php on line 416.
 1 APPPATH/Models/Search.php(416): CodeIgniter\Debug\Exceptions->errorHandler(2, 'A non-numeric value encountered', 'APPPATH/Models/Search.php', 416, [...])
 2 APPPATH/Models/Telegram.php(47): App\Models\Search->addPokazC('142', 'SELECT counter.id, counterId, counterName FROM counter JOIN pokaz ON counter.id = cId
WHERE unit = 131 AND typeC = 8 AND ts != \'2023-02-01\' GROUP BY counter.id', '2023', '02')
 3 APPPATH/Models/Telegram.php(13): App\Models\Telegram->message(Object(stdClass), Object(stdClass), 971483502)
 4 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(Object(stdClass))
 5 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 6 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
 7 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-08 06:09:01 --> A non-numeric value encountered
in APPPATH/Models/Search.php on line 416.
 1 APPPATH/Models/Search.php(416): CodeIgniter\Debug\Exceptions->errorHandler(2, 'A non-numeric value encountered', 'APPPATH/Models/Search.php', 416, [...])
 2 APPPATH/Models/Telegram.php(47): App\Models\Search->addPokazC('142', 'SELECT counter.id, counterId, counterName FROM counter JOIN pokaz ON counter.id = cId
WHERE unit = 131 AND typeC = 8 AND ts != \'2023-02-01\' GROUP BY counter.id', '2023', '02')
 3 APPPATH/Models/Telegram.php(13): App\Models\Telegram->message(Object(stdClass), Object(stdClass), 971483502)
 4 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(Object(stdClass))
 5 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 6 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
 7 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-08 06:11:38 --> A non-numeric value encountered
in APPPATH/Models/Search.php on line 416.
 1 APPPATH/Models/Search.php(416): CodeIgniter\Debug\Exceptions->errorHandler(2, 'A non-numeric value encountered', 'APPPATH/Models/Search.php', 416, [...])
 2 APPPATH/Models/Telegram.php(47): App\Models\Search->addPokazC('142', 'SELECT counter.id, counterId, counterName FROM counter JOIN pokaz ON counter.id = cId
WHERE unit = 131 AND typeC = 8 AND ts != \'2023-02-01\' GROUP BY counter.id', '2023', '02')
 3 APPPATH/Models/Telegram.php(13): App\Models\Telegram->message(Object(stdClass), Object(stdClass), 971483502)
 4 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(Object(stdClass))
 5 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 6 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
 7 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-08 06:13:43 --> mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '2023-02-01' GROUP BY counter.id"}' WHERE telegramChatId = '971483502'' at line 1 in /ext/WWW/konex.com.ua/zayavki/system/Database/MySQLi/Connection.php:292
Stack trace:
#0 /ext/WWW/konex.com.ua/zayavki/system/Database/MySQLi/Connection.php(292): mysqli->query('UPDATE user SET...', 0)
#1 /ext/WWW/konex.com.ua/zayavki/system/Database/BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE user SET...')
#2 /ext/WWW/konex.com.ua/zayavki/system/Database/BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE user SET...')
#3 /ext/WWW/konex.com.ua/zayavki/app/Models/User.php(141): CodeIgniter\Database\BaseConnection->query('UPDATE user SET...')
#4 /ext/WWW/konex.com.ua/zayavki/app/Models/Telegram.php(48): App\Models\User->insertTelegramDataByChatId(971483502, 'confirm', '{"counterPk":"1...')
#5 /ext/WWW/konex.com.ua/zayavki/app/Models/Telegram.php(13): App\Models\Telegram->message(Object(stdClass), Object(stdClass), 971483502)
#6 /ext/WWW/konex.com.ua/zayavki/app/Controllers/Meters.php(158): App\Models\Telegram->telegram(Object(stdClass))
#7 /ext/WWW/konex.com.ua/zayavki/system/CodeIgniter.php(927): App\Controllers\Meters->telegram()
#8 /ext/WWW/konex.com.ua/zayavki/system/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
#9 /ext/WWW/konex.com.ua/zayavki/system/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 /ext/WWW/konex.com.ua/zayavki/public/index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-03-08 06:13:43 --> You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '2023-02-01' GROUP BY counter.id"}' WHERE telegramChatId = '971483502'' at line 1
in SYSTEMPATH/Database/MySQLi/Connection.php on line 292.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(292): mysqli->query('UPDATE user SET telegramState = \'confirm\', telegramMetadata = \'{"counterPk":"142","pokaz":"SELECT counter.id, counterId, counterName FROM counter JOIN pokaz ON counter.id = cId\\nWHERE unit = 131 AND typeC = 8 AND ts != \'2023-02-01\' GROUP BY counter.id"}\' WHERE telegramChatId = \'971483502\'', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE user SET telegramState = \'confirm\', telegramMetadata = \'{"counterPk":"142","pokaz":"SELECT counter.id, counterId, counterName FROM counter JOIN pokaz ON counter.id = cId\\nWHERE unit = 131 AND typeC = 8 AND ts != \'2023-02-01\' GROUP BY counter.id"}\' WHERE telegramChatId = \'971483502\'')
 3 SYSTEMPATH/Database/BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE user SET telegramState = \'confirm\', telegramMetadata = \'{"counterPk":"142","pokaz":"SELECT counter.id, counterId, counterName FROM counter JOIN pokaz ON counter.id = cId\\nWHERE unit = 131 AND typeC = 8 AND ts != \'2023-02-01\' GROUP BY counter.id"}\' WHERE telegramChatId = \'971483502\'')
 4 APPPATH/Models/User.php(141): CodeIgniter\Database\BaseConnection->query('UPDATE user SET telegramState = \'confirm\', telegramMetadata = \'{"counterPk":"142","pokaz":"SELECT counter.id, counterId, counterName FROM counter JOIN pokaz ON counter.id = cId\\nWHERE unit = 131 AND typeC = 8 AND ts != \'2023-02-01\' GROUP BY counter.id"}\' WHERE telegramChatId = \'971483502\'')
 5 APPPATH/Models/Telegram.php(48): App\Models\User->insertTelegramDataByChatId(971483502, 'confirm', '{"counterPk":"142","pokaz":"SELECT counter.id, counterId, counterName FROM counter JOIN pokaz ON counter.id = cId\\nWHERE unit = 131 AND typeC = 8 AND ts != \'2023-02-01\' GROUP BY counter.id"}')
 6 APPPATH/Models/Telegram.php(13): App\Models\Telegram->message(Object(stdClass), Object(stdClass), 971483502)
 7 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(Object(stdClass))
 8 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 9 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
10 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-08 06:16:20 --> mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '2023-02-01' GROUP BY counter.id"}' WHERE telegramChatId = '971483502'' at line 1 in /ext/WWW/konex.com.ua/zayavki/system/Database/MySQLi/Connection.php:292
Stack trace:
#0 /ext/WWW/konex.com.ua/zayavki/system/Database/MySQLi/Connection.php(292): mysqli->query('UPDATE user SET...', 0)
#1 /ext/WWW/konex.com.ua/zayavki/system/Database/BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE user SET...')
#2 /ext/WWW/konex.com.ua/zayavki/system/Database/BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE user SET...')
#3 /ext/WWW/konex.com.ua/zayavki/app/Models/User.php(141): CodeIgniter\Database\BaseConnection->query('UPDATE user SET...')
#4 /ext/WWW/konex.com.ua/zayavki/app/Models/Telegram.php(48): App\Models\User->insertTelegramDataByChatId(971483502, 'confirm', '{"counterPk":"1...')
#5 /ext/WWW/konex.com.ua/zayavki/app/Models/Telegram.php(13): App\Models\Telegram->message(Object(stdClass), Object(stdClass), 971483502)
#6 /ext/WWW/konex.com.ua/zayavki/app/Controllers/Meters.php(158): App\Models\Telegram->telegram(Object(stdClass))
#7 /ext/WWW/konex.com.ua/zayavki/system/CodeIgniter.php(927): App\Controllers\Meters->telegram()
#8 /ext/WWW/konex.com.ua/zayavki/system/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
#9 /ext/WWW/konex.com.ua/zayavki/system/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 /ext/WWW/konex.com.ua/zayavki/public/index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-03-08 06:16:20 --> You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '2023-02-01' GROUP BY counter.id"}' WHERE telegramChatId = '971483502'' at line 1
in SYSTEMPATH/Database/MySQLi/Connection.php on line 292.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(292): mysqli->query('UPDATE user SET telegramState = \'confirm\', telegramMetadata = \'{"counterPk":"142","pokaz":"SELECT counter.id, counterId, counterName FROM counter JOIN pokaz ON counter.id = cId\\nWHERE unit = 131 AND typeC = 8 AND ts != \'2023-02-01\' GROUP BY counter.id"}\' WHERE telegramChatId = \'971483502\'', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE user SET telegramState = \'confirm\', telegramMetadata = \'{"counterPk":"142","pokaz":"SELECT counter.id, counterId, counterName FROM counter JOIN pokaz ON counter.id = cId\\nWHERE unit = 131 AND typeC = 8 AND ts != \'2023-02-01\' GROUP BY counter.id"}\' WHERE telegramChatId = \'971483502\'')
 3 SYSTEMPATH/Database/BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE user SET telegramState = \'confirm\', telegramMetadata = \'{"counterPk":"142","pokaz":"SELECT counter.id, counterId, counterName FROM counter JOIN pokaz ON counter.id = cId\\nWHERE unit = 131 AND typeC = 8 AND ts != \'2023-02-01\' GROUP BY counter.id"}\' WHERE telegramChatId = \'971483502\'')
 4 APPPATH/Models/User.php(141): CodeIgniter\Database\BaseConnection->query('UPDATE user SET telegramState = \'confirm\', telegramMetadata = \'{"counterPk":"142","pokaz":"SELECT counter.id, counterId, counterName FROM counter JOIN pokaz ON counter.id = cId\\nWHERE unit = 131 AND typeC = 8 AND ts != \'2023-02-01\' GROUP BY counter.id"}\' WHERE telegramChatId = \'971483502\'')
 5 APPPATH/Models/Telegram.php(48): App\Models\User->insertTelegramDataByChatId(971483502, 'confirm', '{"counterPk":"142","pokaz":"SELECT counter.id, counterId, counterName FROM counter JOIN pokaz ON counter.id = cId\\nWHERE unit = 131 AND typeC = 8 AND ts != \'2023-02-01\' GROUP BY counter.id"}')
 6 APPPATH/Models/Telegram.php(13): App\Models\Telegram->message(Object(stdClass), Object(stdClass), 971483502)
 7 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(Object(stdClass))
 8 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 9 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
10 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-08 06:18:28 --> mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '2023-02-01' GROUP BY counter.id"}' WHERE telegramChatId = '971483502'' at line 1 in /ext/WWW/konex.com.ua/zayavki/system/Database/MySQLi/Connection.php:292
Stack trace:
#0 /ext/WWW/konex.com.ua/zayavki/system/Database/MySQLi/Connection.php(292): mysqli->query('UPDATE user SET...', 0)
#1 /ext/WWW/konex.com.ua/zayavki/system/Database/BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE user SET...')
#2 /ext/WWW/konex.com.ua/zayavki/system/Database/BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE user SET...')
#3 /ext/WWW/konex.com.ua/zayavki/app/Models/User.php(141): CodeIgniter\Database\BaseConnection->query('UPDATE user SET...')
#4 /ext/WWW/konex.com.ua/zayavki/app/Models/Telegram.php(48): App\Models\User->insertTelegramDataByChatId(971483502, 'confirm', '{"counterPk":"1...')
#5 /ext/WWW/konex.com.ua/zayavki/app/Models/Telegram.php(13): App\Models\Telegram->message(Object(stdClass), Object(stdClass), 971483502)
#6 /ext/WWW/konex.com.ua/zayavki/app/Controllers/Meters.php(158): App\Models\Telegram->telegram(Object(stdClass))
#7 /ext/WWW/konex.com.ua/zayavki/system/CodeIgniter.php(927): App\Controllers\Meters->telegram()
#8 /ext/WWW/konex.com.ua/zayavki/system/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
#9 /ext/WWW/konex.com.ua/zayavki/system/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 /ext/WWW/konex.com.ua/zayavki/public/index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-03-08 06:18:28 --> You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '2023-02-01' GROUP BY counter.id"}' WHERE telegramChatId = '971483502'' at line 1
in SYSTEMPATH/Database/MySQLi/Connection.php on line 292.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(292): mysqli->query('UPDATE user SET telegramState = \'confirm\', telegramMetadata = \'{"counterPk":"142","pokaz":"SELECT counter.id, counterId, counterName FROM counter JOIN pokaz ON counter.id = cId\\nWHERE unit = 131 AND typeC = 8 AND ts != \'2023-02-01\' GROUP BY counter.id"}\' WHERE telegramChatId = \'971483502\'', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE user SET telegramState = \'confirm\', telegramMetadata = \'{"counterPk":"142","pokaz":"SELECT counter.id, counterId, counterName FROM counter JOIN pokaz ON counter.id = cId\\nWHERE unit = 131 AND typeC = 8 AND ts != \'2023-02-01\' GROUP BY counter.id"}\' WHERE telegramChatId = \'971483502\'')
 3 SYSTEMPATH/Database/BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE user SET telegramState = \'confirm\', telegramMetadata = \'{"counterPk":"142","pokaz":"SELECT counter.id, counterId, counterName FROM counter JOIN pokaz ON counter.id = cId\\nWHERE unit = 131 AND typeC = 8 AND ts != \'2023-02-01\' GROUP BY counter.id"}\' WHERE telegramChatId = \'971483502\'')
 4 APPPATH/Models/User.php(141): CodeIgniter\Database\BaseConnection->query('UPDATE user SET telegramState = \'confirm\', telegramMetadata = \'{"counterPk":"142","pokaz":"SELECT counter.id, counterId, counterName FROM counter JOIN pokaz ON counter.id = cId\\nWHERE unit = 131 AND typeC = 8 AND ts != \'2023-02-01\' GROUP BY counter.id"}\' WHERE telegramChatId = \'971483502\'')
 5 APPPATH/Models/Telegram.php(48): App\Models\User->insertTelegramDataByChatId(971483502, 'confirm', '{"counterPk":"142","pokaz":"SELECT counter.id, counterId, counterName FROM counter JOIN pokaz ON counter.id = cId\\nWHERE unit = 131 AND typeC = 8 AND ts != \'2023-02-01\' GROUP BY counter.id"}')
 6 APPPATH/Models/Telegram.php(13): App\Models\Telegram->message(Object(stdClass), Object(stdClass), 971483502)
 7 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(Object(stdClass))
 8 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 9 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
10 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-08 06:21:15 --> mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '2023-02-01' GROUP BY counter.id"}' WHERE telegramChatId = '971483502'' at line 1 in /ext/WWW/konex.com.ua/zayavki/system/Database/MySQLi/Connection.php:292
Stack trace:
#0 /ext/WWW/konex.com.ua/zayavki/system/Database/MySQLi/Connection.php(292): mysqli->query('UPDATE user SET...', 0)
#1 /ext/WWW/konex.com.ua/zayavki/system/Database/BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE user SET...')
#2 /ext/WWW/konex.com.ua/zayavki/system/Database/BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE user SET...')
#3 /ext/WWW/konex.com.ua/zayavki/app/Models/User.php(141): CodeIgniter\Database\BaseConnection->query('UPDATE user SET...')
#4 /ext/WWW/konex.com.ua/zayavki/app/Models/Telegram.php(48): App\Models\User->insertTelegramDataByChatId(971483502, 'confirm', '{"counterPk":"1...')
#5 /ext/WWW/konex.com.ua/zayavki/app/Models/Telegram.php(13): App\Models\Telegram->message(Object(stdClass), Object(stdClass), 971483502)
#6 /ext/WWW/konex.com.ua/zayavki/app/Controllers/Meters.php(158): App\Models\Telegram->telegram(Object(stdClass))
#7 /ext/WWW/konex.com.ua/zayavki/system/CodeIgniter.php(927): App\Controllers\Meters->telegram()
#8 /ext/WWW/konex.com.ua/zayavki/system/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
#9 /ext/WWW/konex.com.ua/zayavki/system/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 /ext/WWW/konex.com.ua/zayavki/public/index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-03-08 06:21:15 --> You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '2023-02-01' GROUP BY counter.id"}' WHERE telegramChatId = '971483502'' at line 1
in SYSTEMPATH/Database/MySQLi/Connection.php on line 292.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(292): mysqli->query('UPDATE user SET telegramState = \'confirm\', telegramMetadata = \'{"counterPk":"142","pokaz":"SELECT counter.id, counterId, counterName FROM counter JOIN pokaz ON counter.id = cId\\nWHERE unit = 131 AND typeC = 8 AND ts != \'2023-02-01\' GROUP BY counter.id"}\' WHERE telegramChatId = \'971483502\'', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE user SET telegramState = \'confirm\', telegramMetadata = \'{"counterPk":"142","pokaz":"SELECT counter.id, counterId, counterName FROM counter JOIN pokaz ON counter.id = cId\\nWHERE unit = 131 AND typeC = 8 AND ts != \'2023-02-01\' GROUP BY counter.id"}\' WHERE telegramChatId = \'971483502\'')
 3 SYSTEMPATH/Database/BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE user SET telegramState = \'confirm\', telegramMetadata = \'{"counterPk":"142","pokaz":"SELECT counter.id, counterId, counterName FROM counter JOIN pokaz ON counter.id = cId\\nWHERE unit = 131 AND typeC = 8 AND ts != \'2023-02-01\' GROUP BY counter.id"}\' WHERE telegramChatId = \'971483502\'')
 4 APPPATH/Models/User.php(141): CodeIgniter\Database\BaseConnection->query('UPDATE user SET telegramState = \'confirm\', telegramMetadata = \'{"counterPk":"142","pokaz":"SELECT counter.id, counterId, counterName FROM counter JOIN pokaz ON counter.id = cId\\nWHERE unit = 131 AND typeC = 8 AND ts != \'2023-02-01\' GROUP BY counter.id"}\' WHERE telegramChatId = \'971483502\'')
 5 APPPATH/Models/Telegram.php(48): App\Models\User->insertTelegramDataByChatId(971483502, 'confirm', '{"counterPk":"142","pokaz":"SELECT counter.id, counterId, counterName FROM counter JOIN pokaz ON counter.id = cId\\nWHERE unit = 131 AND typeC = 8 AND ts != \'2023-02-01\' GROUP BY counter.id"}')
 6 APPPATH/Models/Telegram.php(13): App\Models\Telegram->message(Object(stdClass), Object(stdClass), 971483502)
 7 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(Object(stdClass))
 8 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 9 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
10 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-08 06:24:05 --> syntax error, unexpected 'else' (T_ELSE)
in APPPATH/Models/Telegram.php on line 112.
 1 SYSTEMPATH/Autoloader/Autoloader.php(259): CodeIgniter\Autoloader\Autoloader->includeFile('APPPATH/Models/Telegram.php')
 2 SYSTEMPATH/Autoloader/Autoloader.php(237): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Models\\Telegram')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Models\\Telegram')
 4 APPPATH/Controllers/Meters.php(26): spl_autoload_call('App\\Models\\Telegram')
 5 SYSTEMPATH/CodeIgniter.php(889): App\Controllers\Meters->__construct()
 6 SYSTEMPATH/CodeIgniter.php(473): CodeIgniter\CodeIgniter->createController()
 7 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-08 06:26:12 --> syntax error, unexpected 'else' (T_ELSE)
in APPPATH/Models/Telegram.php on line 113.
 1 SYSTEMPATH/Autoloader/Autoloader.php(259): CodeIgniter\Autoloader\Autoloader->includeFile('APPPATH/Models/Telegram.php')
 2 SYSTEMPATH/Autoloader/Autoloader.php(237): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Models\\Telegram')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Models\\Telegram')
 4 APPPATH/Controllers/Meters.php(26): spl_autoload_call('App\\Models\\Telegram')
 5 SYSTEMPATH/CodeIgniter.php(889): App\Controllers\Meters->__construct()
 6 SYSTEMPATH/CodeIgniter.php(473): CodeIgniter\CodeIgniter->createController()
 7 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-08 06:28:40 --> mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '2023-02-01' GROUP BY counter.id"}' WHERE telegramChatId = '971483502'' at line 1 in /ext/WWW/konex.com.ua/zayavki/system/Database/MySQLi/Connection.php:292
Stack trace:
#0 /ext/WWW/konex.com.ua/zayavki/system/Database/MySQLi/Connection.php(292): mysqli->query('UPDATE user SET...', 0)
#1 /ext/WWW/konex.com.ua/zayavki/system/Database/BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE user SET...')
#2 /ext/WWW/konex.com.ua/zayavki/system/Database/BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE user SET...')
#3 /ext/WWW/konex.com.ua/zayavki/app/Models/User.php(141): CodeIgniter\Database\BaseConnection->query('UPDATE user SET...')
#4 /ext/WWW/konex.com.ua/zayavki/app/Models/Telegram.php(48): App\Models\User->insertTelegramDataByChatId(971483502, 'confirm', '{"counterPk":"1...')
#5 /ext/WWW/konex.com.ua/zayavki/app/Models/Telegram.php(13): App\Models\Telegram->message(Object(stdClass), Object(stdClass), 971483502)
#6 /ext/WWW/konex.com.ua/zayavki/app/Controllers/Meters.php(158): App\Models\Telegram->telegram(Object(stdClass))
#7 /ext/WWW/konex.com.ua/zayavki/system/CodeIgniter.php(927): App\Controllers\Meters->telegram()
#8 /ext/WWW/konex.com.ua/zayavki/system/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
#9 /ext/WWW/konex.com.ua/zayavki/system/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 /ext/WWW/konex.com.ua/zayavki/public/index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-03-08 06:28:40 --> You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '2023-02-01' GROUP BY counter.id"}' WHERE telegramChatId = '971483502'' at line 1
in SYSTEMPATH/Database/MySQLi/Connection.php on line 292.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(292): mysqli->query('UPDATE user SET telegramState = \'confirm\', telegramMetadata = \'{"counterPk":"142","pokaz":"SELECT counter.id, counterId, counterName FROM counter JOIN pokaz ON counter.id = cId\\nWHERE unit = 131 AND typeC = 8 AND ts != \'2023-02-01\' GROUP BY counter.id"}\' WHERE telegramChatId = \'971483502\'', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE user SET telegramState = \'confirm\', telegramMetadata = \'{"counterPk":"142","pokaz":"SELECT counter.id, counterId, counterName FROM counter JOIN pokaz ON counter.id = cId\\nWHERE unit = 131 AND typeC = 8 AND ts != \'2023-02-01\' GROUP BY counter.id"}\' WHERE telegramChatId = \'971483502\'')
 3 SYSTEMPATH/Database/BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE user SET telegramState = \'confirm\', telegramMetadata = \'{"counterPk":"142","pokaz":"SELECT counter.id, counterId, counterName FROM counter JOIN pokaz ON counter.id = cId\\nWHERE unit = 131 AND typeC = 8 AND ts != \'2023-02-01\' GROUP BY counter.id"}\' WHERE telegramChatId = \'971483502\'')
 4 APPPATH/Models/User.php(141): CodeIgniter\Database\BaseConnection->query('UPDATE user SET telegramState = \'confirm\', telegramMetadata = \'{"counterPk":"142","pokaz":"SELECT counter.id, counterId, counterName FROM counter JOIN pokaz ON counter.id = cId\\nWHERE unit = 131 AND typeC = 8 AND ts != \'2023-02-01\' GROUP BY counter.id"}\' WHERE telegramChatId = \'971483502\'')
 5 APPPATH/Models/Telegram.php(48): App\Models\User->insertTelegramDataByChatId(971483502, 'confirm', '{"counterPk":"142","pokaz":"SELECT counter.id, counterId, counterName FROM counter JOIN pokaz ON counter.id = cId\\nWHERE unit = 131 AND typeC = 8 AND ts != \'2023-02-01\' GROUP BY counter.id"}')
 6 APPPATH/Models/Telegram.php(13): App\Models\Telegram->message(Object(stdClass), Object(stdClass), 971483502)
 7 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(Object(stdClass))
 8 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 9 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
10 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-08 06:31:12 --> mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '2023-02-01' GROUP BY counter.id"}' WHERE telegramChatId = '971483502'' at line 1 in /ext/WWW/konex.com.ua/zayavki/system/Database/MySQLi/Connection.php:292
Stack trace:
#0 /ext/WWW/konex.com.ua/zayavki/system/Database/MySQLi/Connection.php(292): mysqli->query('UPDATE user SET...', 0)
#1 /ext/WWW/konex.com.ua/zayavki/system/Database/BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE user SET...')
#2 /ext/WWW/konex.com.ua/zayavki/system/Database/BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE user SET...')
#3 /ext/WWW/konex.com.ua/zayavki/app/Models/User.php(141): CodeIgniter\Database\BaseConnection->query('UPDATE user SET...')
#4 /ext/WWW/konex.com.ua/zayavki/app/Models/Telegram.php(48): App\Models\User->insertTelegramDataByChatId(971483502, 'confirm', '{"counterPk":"1...')
#5 /ext/WWW/konex.com.ua/zayavki/app/Models/Telegram.php(13): App\Models\Telegram->message(Object(stdClass), Object(stdClass), 971483502)
#6 /ext/WWW/konex.com.ua/zayavki/app/Controllers/Meters.php(158): App\Models\Telegram->telegram(Object(stdClass))
#7 /ext/WWW/konex.com.ua/zayavki/system/CodeIgniter.php(927): App\Controllers\Meters->telegram()
#8 /ext/WWW/konex.com.ua/zayavki/system/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
#9 /ext/WWW/konex.com.ua/zayavki/system/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 /ext/WWW/konex.com.ua/zayavki/public/index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-03-08 06:31:12 --> You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '2023-02-01' GROUP BY counter.id"}' WHERE telegramChatId = '971483502'' at line 1
in SYSTEMPATH/Database/MySQLi/Connection.php on line 292.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(292): mysqli->query('UPDATE user SET telegramState = \'confirm\', telegramMetadata = \'{"counterPk":"142","pokaz":"SELECT counter.id, counterId, counterName FROM counter JOIN pokaz ON counter.id = cId\\nWHERE unit = 131 AND typeC = 8 AND ts != \'2023-02-01\' GROUP BY counter.id"}\' WHERE telegramChatId = \'971483502\'', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE user SET telegramState = \'confirm\', telegramMetadata = \'{"counterPk":"142","pokaz":"SELECT counter.id, counterId, counterName FROM counter JOIN pokaz ON counter.id = cId\\nWHERE unit = 131 AND typeC = 8 AND ts != \'2023-02-01\' GROUP BY counter.id"}\' WHERE telegramChatId = \'971483502\'')
 3 SYSTEMPATH/Database/BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE user SET telegramState = \'confirm\', telegramMetadata = \'{"counterPk":"142","pokaz":"SELECT counter.id, counterId, counterName FROM counter JOIN pokaz ON counter.id = cId\\nWHERE unit = 131 AND typeC = 8 AND ts != \'2023-02-01\' GROUP BY counter.id"}\' WHERE telegramChatId = \'971483502\'')
 4 APPPATH/Models/User.php(141): CodeIgniter\Database\BaseConnection->query('UPDATE user SET telegramState = \'confirm\', telegramMetadata = \'{"counterPk":"142","pokaz":"SELECT counter.id, counterId, counterName FROM counter JOIN pokaz ON counter.id = cId\\nWHERE unit = 131 AND typeC = 8 AND ts != \'2023-02-01\' GROUP BY counter.id"}\' WHERE telegramChatId = \'971483502\'')
 5 APPPATH/Models/Telegram.php(48): App\Models\User->insertTelegramDataByChatId(971483502, 'confirm', '{"counterPk":"142","pokaz":"SELECT counter.id, counterId, counterName FROM counter JOIN pokaz ON counter.id = cId\\nWHERE unit = 131 AND typeC = 8 AND ts != \'2023-02-01\' GROUP BY counter.id"}')
 6 APPPATH/Models/Telegram.php(13): App\Models\Telegram->message(Object(stdClass), Object(stdClass), 971483502)
 7 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(Object(stdClass))
 8 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 9 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
10 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-08 06:33:44 --> mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '2023-02-01' GROUP BY counter.id"}' WHERE telegramChatId = '971483502'' at line 1 in /ext/WWW/konex.com.ua/zayavki/system/Database/MySQLi/Connection.php:292
Stack trace:
#0 /ext/WWW/konex.com.ua/zayavki/system/Database/MySQLi/Connection.php(292): mysqli->query('UPDATE user SET...', 0)
#1 /ext/WWW/konex.com.ua/zayavki/system/Database/BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE user SET...')
#2 /ext/WWW/konex.com.ua/zayavki/system/Database/BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE user SET...')
#3 /ext/WWW/konex.com.ua/zayavki/app/Models/User.php(141): CodeIgniter\Database\BaseConnection->query('UPDATE user SET...')
#4 /ext/WWW/konex.com.ua/zayavki/app/Models/Telegram.php(48): App\Models\User->insertTelegramDataByChatId(971483502, 'confirm', '{"counterPk":"1...')
#5 /ext/WWW/konex.com.ua/zayavki/app/Models/Telegram.php(13): App\Models\Telegram->message(Object(stdClass), Object(stdClass), 971483502)
#6 /ext/WWW/konex.com.ua/zayavki/app/Controllers/Meters.php(158): App\Models\Telegram->telegram(Object(stdClass))
#7 /ext/WWW/konex.com.ua/zayavki/system/CodeIgniter.php(927): App\Controllers\Meters->telegram()
#8 /ext/WWW/konex.com.ua/zayavki/system/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
#9 /ext/WWW/konex.com.ua/zayavki/system/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 /ext/WWW/konex.com.ua/zayavki/public/index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-03-08 06:33:44 --> You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '2023-02-01' GROUP BY counter.id"}' WHERE telegramChatId = '971483502'' at line 1
in SYSTEMPATH/Database/MySQLi/Connection.php on line 292.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(292): mysqli->query('UPDATE user SET telegramState = \'confirm\', telegramMetadata = \'{"counterPk":"142","pokaz":"SELECT counter.id, counterId, counterName FROM counter JOIN pokaz ON counter.id = cId\\nWHERE unit = 131 AND typeC = 8 AND ts != \'2023-02-01\' GROUP BY counter.id"}\' WHERE telegramChatId = \'971483502\'', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE user SET telegramState = \'confirm\', telegramMetadata = \'{"counterPk":"142","pokaz":"SELECT counter.id, counterId, counterName FROM counter JOIN pokaz ON counter.id = cId\\nWHERE unit = 131 AND typeC = 8 AND ts != \'2023-02-01\' GROUP BY counter.id"}\' WHERE telegramChatId = \'971483502\'')
 3 SYSTEMPATH/Database/BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE user SET telegramState = \'confirm\', telegramMetadata = \'{"counterPk":"142","pokaz":"SELECT counter.id, counterId, counterName FROM counter JOIN pokaz ON counter.id = cId\\nWHERE unit = 131 AND typeC = 8 AND ts != \'2023-02-01\' GROUP BY counter.id"}\' WHERE telegramChatId = \'971483502\'')
 4 APPPATH/Models/User.php(141): CodeIgniter\Database\BaseConnection->query('UPDATE user SET telegramState = \'confirm\', telegramMetadata = \'{"counterPk":"142","pokaz":"SELECT counter.id, counterId, counterName FROM counter JOIN pokaz ON counter.id = cId\\nWHERE unit = 131 AND typeC = 8 AND ts != \'2023-02-01\' GROUP BY counter.id"}\' WHERE telegramChatId = \'971483502\'')
 5 APPPATH/Models/Telegram.php(48): App\Models\User->insertTelegramDataByChatId(971483502, 'confirm', '{"counterPk":"142","pokaz":"SELECT counter.id, counterId, counterName FROM counter JOIN pokaz ON counter.id = cId\\nWHERE unit = 131 AND typeC = 8 AND ts != \'2023-02-01\' GROUP BY counter.id"}')
 6 APPPATH/Models/Telegram.php(13): App\Models\Telegram->message(Object(stdClass), Object(stdClass), 971483502)
 7 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(Object(stdClass))
 8 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 9 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
10 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-08 06:36:25 --> Trying to get property 'counterPk' of non-object
in APPPATH/Models/Telegram.php on line 46.
 1 APPPATH/Models/Telegram.php(46): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get property \'counterPk\' of non-object', 'APPPATH/Models/Telegram.php', 46, [...])
 2 APPPATH/Models/Telegram.php(13): App\Models\Telegram->message(Object(stdClass), Object(stdClass), 971483502)
 3 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(Object(stdClass))
 4 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 5 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
 6 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-08 06:45:31 --> syntax error, unexpected 'case' (T_CASE)
in APPPATH/Models/Telegram.php on line 33.
 1 SYSTEMPATH/Autoloader/Autoloader.php(259): CodeIgniter\Autoloader\Autoloader->includeFile('APPPATH/Models/Telegram.php')
 2 SYSTEMPATH/Autoloader/Autoloader.php(237): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Models\\Telegram')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Models\\Telegram')
 4 APPPATH/Controllers/Meters.php(26): spl_autoload_call('App\\Models\\Telegram')
 5 SYSTEMPATH/CodeIgniter.php(889): App\Controllers\Meters->__construct()
 6 SYSTEMPATH/CodeIgniter.php(473): CodeIgniter\CodeIgniter->createController()
 7 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-08 06:46:44 --> Trying to get property 'counterPk' of non-object
in APPPATH/Models/Telegram.php on line 46.
 1 APPPATH/Models/Telegram.php(46): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get property \'counterPk\' of non-object', 'APPPATH/Models/Telegram.php', 46, [...])
 2 APPPATH/Models/Telegram.php(13): App\Models\Telegram->message(Object(stdClass), Object(stdClass), 971483502)
 3 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(Object(stdClass))
 4 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 5 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
 6 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-08 06:49:28 --> Trying to get property 'counterPk' of non-object
in APPPATH/Models/Telegram.php on line 46.
 1 APPPATH/Models/Telegram.php(46): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get property \'counterPk\' of non-object', 'APPPATH/Models/Telegram.php', 46, [...])
 2 APPPATH/Models/Telegram.php(13): App\Models\Telegram->message(Object(stdClass), Object(stdClass), 971483502)
 3 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(Object(stdClass))
 4 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 5 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
 6 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-08 06:49:35 --> Trying to get property 'counterPk' of non-object
in APPPATH/Models/Telegram.php on line 46.
 1 APPPATH/Models/Telegram.php(46): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get property \'counterPk\' of non-object', 'APPPATH/Models/Telegram.php', 46, [...])
 2 APPPATH/Models/Telegram.php(13): App\Models\Telegram->message(Object(stdClass), Object(stdClass), 971483502)
 3 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(Object(stdClass))
 4 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 5 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
 6 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-08 06:51:06 --> Trying to get property 'counterPk' of non-object
in APPPATH/Models/Telegram.php on line 46.
 1 APPPATH/Models/Telegram.php(46): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get property \'counterPk\' of non-object', 'APPPATH/Models/Telegram.php', 46, [...])
 2 APPPATH/Models/Telegram.php(13): App\Models\Telegram->message(Object(stdClass), Object(stdClass), 971483502)
 3 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(Object(stdClass))
 4 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 5 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
 6 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-08 06:52:58 --> mysqli_sql_exception: Unknown column 'electricity' in 'where clause' in /ext/WWW/konex.com.ua/zayavki/system/Database/MySQLi/Connection.php:292
Stack trace:
#0 /ext/WWW/konex.com.ua/zayavki/system/Database/MySQLi/Connection.php(292): mysqli->query('SELECT pokaz.in...', 0)
#1 /ext/WWW/konex.com.ua/zayavki/system/Database/BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('SELECT pokaz.in...')
#2 /ext/WWW/konex.com.ua/zayavki/system/Database/BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT pokaz.in...')
#3 /ext/WWW/konex.com.ua/zayavki/app/Models/Search.php(300): CodeIgniter\Database\BaseConnection->query('SELECT pokaz.in...')
#4 /ext/WWW/konex.com.ua/zayavki/app/Models/Telegram.php(96): App\Models\Search->getLastPokazByCounter('electricity')
#5 /ext/WWW/konex.com.ua/zayavki/app/Models/Telegram.php(13): App\Models\Telegram->callback(Object(stdClass), Object(stdClass), 971483502)
#6 /ext/WWW/konex.com.ua/zayavki/app/Controllers/Meters.php(158): App\Models\Telegram->telegram(Object(stdClass))
#7 /ext/WWW/konex.com.ua/zayavki/system/CodeIgniter.php(927): App\Controllers\Meters->telegram()
#8 /ext/WWW/konex.com.ua/zayavki/system/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
#9 /ext/WWW/konex.com.ua/zayavki/system/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 /ext/WWW/konex.com.ua/zayavki/public/index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-03-08 06:52:58 --> Unknown column 'electricity' in 'where clause'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 292.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(292): mysqli->query('SELECT pokaz.index FROM pokaz WHERE cId = electricity AND ts < \'2023-02-01\' ORDER BY ts DESC LIMIT 1', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('SELECT pokaz.index FROM pokaz WHERE cId = electricity AND ts < \'2023-02-01\' ORDER BY ts DESC LIMIT 1')
 3 SYSTEMPATH/Database/BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT pokaz.index FROM pokaz WHERE cId = electricity AND ts < \'2023-02-01\' ORDER BY ts DESC LIMIT 1')
 4 APPPATH/Models/Search.php(300): CodeIgniter\Database\BaseConnection->query('SELECT pokaz.index FROM pokaz WHERE cId = electricity AND ts < \'2023-02-01\' ORDER BY ts DESC LIMIT 1')
 5 APPPATH/Models/Telegram.php(96): App\Models\Search->getLastPokazByCounter('electricity')
 6 APPPATH/Models/Telegram.php(13): App\Models\Telegram->callback(Object(stdClass), Object(stdClass), 971483502)
 7 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(Object(stdClass))
 8 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 9 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
10 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-08 06:54:05 --> mysqli_sql_exception: Unknown column 'electricity' in 'where clause' in /ext/WWW/konex.com.ua/zayavki/system/Database/MySQLi/Connection.php:292
Stack trace:
#0 /ext/WWW/konex.com.ua/zayavki/system/Database/MySQLi/Connection.php(292): mysqli->query('SELECT count(*)...', 0)
#1 /ext/WWW/konex.com.ua/zayavki/system/Database/BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('SELECT count(*)...')
#2 /ext/WWW/konex.com.ua/zayavki/system/Database/BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT count(*)...')
#3 /ext/WWW/konex.com.ua/zayavki/app/Models/Search.php(393): CodeIgniter\Database\BaseConnection->query('SELECT count(*)...')
#4 /ext/WWW/konex.com.ua/zayavki/app/Models/Telegram.php(47): App\Models\Search->checkPokaz('electricity', '2023-02-01', 'SELECT counter....')
#5 /ext/WWW/konex.com.ua/zayavki/app/Models/Telegram.php(13): App\Models\Telegram->message(Object(stdClass), Object(stdClass), 971483502)
#6 /ext/WWW/konex.com.ua/zayavki/app/Controllers/Meters.php(158): App\Models\Telegram->telegram(Object(stdClass))
#7 /ext/WWW/konex.com.ua/zayavki/system/CodeIgniter.php(927): App\Controllers\Meters->telegram()
#8 /ext/WWW/konex.com.ua/zayavki/system/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
#9 /ext/WWW/konex.com.ua/zayavki/system/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 /ext/WWW/konex.com.ua/zayavki/public/index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-03-08 06:54:05 --> Unknown column 'electricity' in 'where clause'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 292.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(292): mysqli->query('SELECT count(*) AS count FROM pokaz as p WHERE p.cId=electricity AND p.ts>\'2023-02-01\' ORDER BY ts LIMIT 1', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('SELECT count(*) AS count FROM pokaz as p WHERE p.cId=electricity AND p.ts>\'2023-02-01\' ORDER BY ts LIMIT 1')
 3 SYSTEMPATH/Database/BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT count(*) AS count FROM pokaz as p WHERE p.cId=electricity AND p.ts>\'2023-02-01\' ORDER BY ts LIMIT 1')
 4 APPPATH/Models/Search.php(393): CodeIgniter\Database\BaseConnection->query('SELECT count(*) AS count FROM pokaz as p WHERE p.cId=electricity AND p.ts>\'2023-02-01\' ORDER BY ts LIMIT 1')
 5 APPPATH/Models/Telegram.php(47): App\Models\Search->checkPokaz('electricity', '2023-02-01', 'SELECT counter.id, counterId, counterName FROM counter JOIN pokaz ON counter.id = cId
WHERE unit = 131 AND typeC = 8 AND ts != \'2023-02-01\' GROUP BY counter.id')
 6 APPPATH/Models/Telegram.php(13): App\Models\Telegram->message(Object(stdClass), Object(stdClass), 971483502)
 7 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(Object(stdClass))
 8 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 9 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
10 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-08 06:57:06 --> mysqli_sql_exception: Unknown column 'electricity' in 'where clause' in /ext/WWW/konex.com.ua/zayavki/system/Database/MySQLi/Connection.php:292
Stack trace:
#0 /ext/WWW/konex.com.ua/zayavki/system/Database/MySQLi/Connection.php(292): mysqli->query('SELECT count(*)...', 0)
#1 /ext/WWW/konex.com.ua/zayavki/system/Database/BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('SELECT count(*)...')
#2 /ext/WWW/konex.com.ua/zayavki/system/Database/BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT count(*)...')
#3 /ext/WWW/konex.com.ua/zayavki/app/Models/Search.php(393): CodeIgniter\Database\BaseConnection->query('SELECT count(*)...')
#4 /ext/WWW/konex.com.ua/zayavki/app/Models/Telegram.php(47): App\Models\Search->checkPokaz('electricity', '2023-02-01', 'SELECT counter....')
#5 /ext/WWW/konex.com.ua/zayavki/app/Models/Telegram.php(13): App\Models\Telegram->message(Object(stdClass), Object(stdClass), 971483502)
#6 /ext/WWW/konex.com.ua/zayavki/app/Controllers/Meters.php(158): App\Models\Telegram->telegram(Object(stdClass))
#7 /ext/WWW/konex.com.ua/zayavki/system/CodeIgniter.php(927): App\Controllers\Meters->telegram()
#8 /ext/WWW/konex.com.ua/zayavki/system/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
#9 /ext/WWW/konex.com.ua/zayavki/system/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 /ext/WWW/konex.com.ua/zayavki/public/index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-03-08 06:57:06 --> Unknown column 'electricity' in 'where clause'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 292.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(292): mysqli->query('SELECT count(*) AS count FROM pokaz as p WHERE p.cId=electricity AND p.ts>\'2023-02-01\' ORDER BY ts LIMIT 1', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('SELECT count(*) AS count FROM pokaz as p WHERE p.cId=electricity AND p.ts>\'2023-02-01\' ORDER BY ts LIMIT 1')
 3 SYSTEMPATH/Database/BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT count(*) AS count FROM pokaz as p WHERE p.cId=electricity AND p.ts>\'2023-02-01\' ORDER BY ts LIMIT 1')
 4 APPPATH/Models/Search.php(393): CodeIgniter\Database\BaseConnection->query('SELECT count(*) AS count FROM pokaz as p WHERE p.cId=electricity AND p.ts>\'2023-02-01\' ORDER BY ts LIMIT 1')
 5 APPPATH/Models/Telegram.php(47): App\Models\Search->checkPokaz('electricity', '2023-02-01', 'SELECT counter.id, counterId, counterName FROM counter JOIN pokaz ON counter.id = cId
WHERE unit = 131 AND typeC = 8 AND ts != \'2023-02-01\' GROUP BY counter.id')
 6 APPPATH/Models/Telegram.php(13): App\Models\Telegram->message(Object(stdClass), Object(stdClass), 971483502)
 7 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(Object(stdClass))
 8 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 9 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
10 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-08 07:09:32 --> Trying to get property 'counterPk' of non-object
in APPPATH/Models/Telegram.php on line 46.
 1 APPPATH/Models/Telegram.php(46): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get property \'counterPk\' of non-object', 'APPPATH/Models/Telegram.php', 46, [...])
 2 APPPATH/Models/Telegram.php(13): App\Models\Telegram->message(Object(stdClass), Object(stdClass), 971483502)
 3 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(Object(stdClass))
 4 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 5 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
 6 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-08 07:10:02 --> Trying to get property 'counterPk' of non-object
in APPPATH/Models/Telegram.php on line 46.
 1 APPPATH/Models/Telegram.php(46): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get property \'counterPk\' of non-object', 'APPPATH/Models/Telegram.php', 46, [...])
 2 APPPATH/Models/Telegram.php(13): App\Models\Telegram->message(Object(stdClass), Object(stdClass), 971483502)
 3 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(Object(stdClass))
 4 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 5 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
 6 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-08 07:17:04 --> mysqli_sql_exception: Unknown column 'electricity' in 'where clause' in /ext/WWW/konex.com.ua/zayavki/system/Database/MySQLi/Connection.php:292
Stack trace:
#0 /ext/WWW/konex.com.ua/zayavki/system/Database/MySQLi/Connection.php(292): mysqli->query('SELECT pokaz.in...', 0)
#1 /ext/WWW/konex.com.ua/zayavki/system/Database/BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('SELECT pokaz.in...')
#2 /ext/WWW/konex.com.ua/zayavki/system/Database/BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT pokaz.in...')
#3 /ext/WWW/konex.com.ua/zayavki/app/Models/Search.php(300): CodeIgniter\Database\BaseConnection->query('SELECT pokaz.in...')
#4 /ext/WWW/konex.com.ua/zayavki/app/Models/Telegram.php(101): App\Models\Search->getLastPokazByCounter('electricity')
#5 /ext/WWW/konex.com.ua/zayavki/app/Models/Telegram.php(13): App\Models\Telegram->callback(Object(stdClass), Object(stdClass), 971483502)
#6 /ext/WWW/konex.com.ua/zayavki/app/Controllers/Meters.php(158): App\Models\Telegram->telegram(Object(stdClass))
#7 /ext/WWW/konex.com.ua/zayavki/system/CodeIgniter.php(927): App\Controllers\Meters->telegram()
#8 /ext/WWW/konex.com.ua/zayavki/system/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
#9 /ext/WWW/konex.com.ua/zayavki/system/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 /ext/WWW/konex.com.ua/zayavki/public/index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-03-08 07:17:04 --> Unknown column 'electricity' in 'where clause'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 292.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(292): mysqli->query('SELECT pokaz.index FROM pokaz WHERE cId = electricity AND ts < \'2023-02-01\' ORDER BY ts DESC LIMIT 1', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('SELECT pokaz.index FROM pokaz WHERE cId = electricity AND ts < \'2023-02-01\' ORDER BY ts DESC LIMIT 1')
 3 SYSTEMPATH/Database/BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT pokaz.index FROM pokaz WHERE cId = electricity AND ts < \'2023-02-01\' ORDER BY ts DESC LIMIT 1')
 4 APPPATH/Models/Search.php(300): CodeIgniter\Database\BaseConnection->query('SELECT pokaz.index FROM pokaz WHERE cId = electricity AND ts < \'2023-02-01\' ORDER BY ts DESC LIMIT 1')
 5 APPPATH/Models/Telegram.php(101): App\Models\Search->getLastPokazByCounter('electricity')
 6 APPPATH/Models/Telegram.php(13): App\Models\Telegram->callback(Object(stdClass), Object(stdClass), 971483502)
 7 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(Object(stdClass))
 8 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 9 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
10 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-08 07:21:57 --> mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '2023-02-01' GROUP BY counter.id"}' WHERE telegramChatId = '971483502'' at line 1 in /ext/WWW/konex.com.ua/zayavki/system/Database/MySQLi/Connection.php:292
Stack trace:
#0 /ext/WWW/konex.com.ua/zayavki/system/Database/MySQLi/Connection.php(292): mysqli->query('UPDATE user SET...', 0)
#1 /ext/WWW/konex.com.ua/zayavki/system/Database/BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE user SET...')
#2 /ext/WWW/konex.com.ua/zayavki/system/Database/BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE user SET...')
#3 /ext/WWW/konex.com.ua/zayavki/app/Models/User.php(141): CodeIgniter\Database\BaseConnection->query('UPDATE user SET...')
#4 /ext/WWW/konex.com.ua/zayavki/app/Models/Telegram.php(48): App\Models\User->insertTelegramDataByChatId(971483502, 'confirm', '{"counterPk":"1...')
#5 /ext/WWW/konex.com.ua/zayavki/app/Models/Telegram.php(13): App\Models\Telegram->message(Object(stdClass), Object(stdClass), 971483502)
#6 /ext/WWW/konex.com.ua/zayavki/app/Controllers/Meters.php(158): App\Models\Telegram->telegram(Object(stdClass))
#7 /ext/WWW/konex.com.ua/zayavki/system/CodeIgniter.php(927): App\Controllers\Meters->telegram()
#8 /ext/WWW/konex.com.ua/zayavki/system/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
#9 /ext/WWW/konex.com.ua/zayavki/system/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 /ext/WWW/konex.com.ua/zayavki/public/index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-03-08 07:21:57 --> You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '2023-02-01' GROUP BY counter.id"}' WHERE telegramChatId = '971483502'' at line 1
in SYSTEMPATH/Database/MySQLi/Connection.php on line 292.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(292): mysqli->query('UPDATE user SET telegramState = \'confirm\', telegramMetadata = \'{"counterPk":"142","pokaz":"SELECT counter.id, counterId, counterName FROM counter JOIN pokaz ON counter.id = cId\\nWHERE unit = 131 AND typeC = 8 AND ts != \'2023-02-01\' GROUP BY counter.id"}\' WHERE telegramChatId = \'971483502\'', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE user SET telegramState = \'confirm\', telegramMetadata = \'{"counterPk":"142","pokaz":"SELECT counter.id, counterId, counterName FROM counter JOIN pokaz ON counter.id = cId\\nWHERE unit = 131 AND typeC = 8 AND ts != \'2023-02-01\' GROUP BY counter.id"}\' WHERE telegramChatId = \'971483502\'')
 3 SYSTEMPATH/Database/BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE user SET telegramState = \'confirm\', telegramMetadata = \'{"counterPk":"142","pokaz":"SELECT counter.id, counterId, counterName FROM counter JOIN pokaz ON counter.id = cId\\nWHERE unit = 131 AND typeC = 8 AND ts != \'2023-02-01\' GROUP BY counter.id"}\' WHERE telegramChatId = \'971483502\'')
 4 APPPATH/Models/User.php(141): CodeIgniter\Database\BaseConnection->query('UPDATE user SET telegramState = \'confirm\', telegramMetadata = \'{"counterPk":"142","pokaz":"SELECT counter.id, counterId, counterName FROM counter JOIN pokaz ON counter.id = cId\\nWHERE unit = 131 AND typeC = 8 AND ts != \'2023-02-01\' GROUP BY counter.id"}\' WHERE telegramChatId = \'971483502\'')
 5 APPPATH/Models/Telegram.php(48): App\Models\User->insertTelegramDataByChatId(971483502, 'confirm', '{"counterPk":"142","pokaz":"SELECT counter.id, counterId, counterName FROM counter JOIN pokaz ON counter.id = cId\\nWHERE unit = 131 AND typeC = 8 AND ts != \'2023-02-01\' GROUP BY counter.id"}')
 6 APPPATH/Models/Telegram.php(13): App\Models\Telegram->message(Object(stdClass), Object(stdClass), 971483502)
 7 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(Object(stdClass))
 8 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 9 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
10 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-08 07:27:01 --> mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '2023-02-01' GROUP BY counter.id"}' WHERE telegramChatId = '971483502'' at line 1 in /ext/WWW/konex.com.ua/zayavki/system/Database/MySQLi/Connection.php:292
Stack trace:
#0 /ext/WWW/konex.com.ua/zayavki/system/Database/MySQLi/Connection.php(292): mysqli->query('UPDATE user SET...', 0)
#1 /ext/WWW/konex.com.ua/zayavki/system/Database/BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE user SET...')
#2 /ext/WWW/konex.com.ua/zayavki/system/Database/BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE user SET...')
#3 /ext/WWW/konex.com.ua/zayavki/app/Models/User.php(141): CodeIgniter\Database\BaseConnection->query('UPDATE user SET...')
#4 /ext/WWW/konex.com.ua/zayavki/app/Models/Telegram.php(48): App\Models\User->insertTelegramDataByChatId(971483502, 'confirm', '{"counterPk":"1...')
#5 /ext/WWW/konex.com.ua/zayavki/app/Models/Telegram.php(13): App\Models\Telegram->message(Object(stdClass), Object(stdClass), 971483502)
#6 /ext/WWW/konex.com.ua/zayavki/app/Controllers/Meters.php(158): App\Models\Telegram->telegram(Object(stdClass))
#7 /ext/WWW/konex.com.ua/zayavki/system/CodeIgniter.php(927): App\Controllers\Meters->telegram()
#8 /ext/WWW/konex.com.ua/zayavki/system/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
#9 /ext/WWW/konex.com.ua/zayavki/system/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 /ext/WWW/konex.com.ua/zayavki/public/index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-03-08 07:27:01 --> You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '2023-02-01' GROUP BY counter.id"}' WHERE telegramChatId = '971483502'' at line 1
in SYSTEMPATH/Database/MySQLi/Connection.php on line 292.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(292): mysqli->query('UPDATE user SET telegramState = \'confirm\', telegramMetadata = \'{"counterPk":"142","pokaz":"SELECT counter.id, counterId, counterName FROM counter JOIN pokaz ON counter.id = cId\\nWHERE unit = 131 AND typeC = 8 AND ts != \'2023-02-01\' GROUP BY counter.id"}\' WHERE telegramChatId = \'971483502\'', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE user SET telegramState = \'confirm\', telegramMetadata = \'{"counterPk":"142","pokaz":"SELECT counter.id, counterId, counterName FROM counter JOIN pokaz ON counter.id = cId\\nWHERE unit = 131 AND typeC = 8 AND ts != \'2023-02-01\' GROUP BY counter.id"}\' WHERE telegramChatId = \'971483502\'')
 3 SYSTEMPATH/Database/BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE user SET telegramState = \'confirm\', telegramMetadata = \'{"counterPk":"142","pokaz":"SELECT counter.id, counterId, counterName FROM counter JOIN pokaz ON counter.id = cId\\nWHERE unit = 131 AND typeC = 8 AND ts != \'2023-02-01\' GROUP BY counter.id"}\' WHERE telegramChatId = \'971483502\'')
 4 APPPATH/Models/User.php(141): CodeIgniter\Database\BaseConnection->query('UPDATE user SET telegramState = \'confirm\', telegramMetadata = \'{"counterPk":"142","pokaz":"SELECT counter.id, counterId, counterName FROM counter JOIN pokaz ON counter.id = cId\\nWHERE unit = 131 AND typeC = 8 AND ts != \'2023-02-01\' GROUP BY counter.id"}\' WHERE telegramChatId = \'971483502\'')
 5 APPPATH/Models/Telegram.php(48): App\Models\User->insertTelegramDataByChatId(971483502, 'confirm', '{"counterPk":"142","pokaz":"SELECT counter.id, counterId, counterName FROM counter JOIN pokaz ON counter.id = cId\\nWHERE unit = 131 AND typeC = 8 AND ts != \'2023-02-01\' GROUP BY counter.id"}')
 6 APPPATH/Models/Telegram.php(13): App\Models\Telegram->message(Object(stdClass), Object(stdClass), 971483502)
 7 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(Object(stdClass))
 8 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 9 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
10 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-08 07:34:08 --> json_decode() expects parameter 1 to be string, object given
in APPPATH/Models/Telegram.php on line 113.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'json_decode() expects parameter 1 to be string, object given', 'APPPATH/Models/Telegram.php', 113, [...])
 2 APPPATH/Models/Telegram.php(113): json_decode(Object(stdClass))
 3 APPPATH/Models/Telegram.php(13): App\Models\Telegram->callback(Object(stdClass), Object(stdClass), 971483502)
 4 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(Object(stdClass))
 5 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 6 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
 7 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-08 07:34:30 --> json_decode() expects parameter 1 to be string, object given
in APPPATH/Models/Telegram.php on line 113.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'json_decode() expects parameter 1 to be string, object given', 'APPPATH/Models/Telegram.php', 113, [...])
 2 APPPATH/Models/Telegram.php(113): json_decode(Object(stdClass))
 3 APPPATH/Models/Telegram.php(13): App\Models\Telegram->callback(Object(stdClass), Object(stdClass), 971483502)
 4 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(Object(stdClass))
 5 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 6 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
 7 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-08 07:35:42 --> json_decode() expects parameter 1 to be string, object given
in APPPATH/Models/Telegram.php on line 113.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'json_decode() expects parameter 1 to be string, object given', 'APPPATH/Models/Telegram.php', 113, [...])
 2 APPPATH/Models/Telegram.php(113): json_decode(Object(stdClass))
 3 APPPATH/Models/Telegram.php(13): App\Models\Telegram->callback(Object(stdClass), Object(stdClass), 971483502)
 4 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(Object(stdClass))
 5 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 6 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
 7 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-08 07:37:02 --> Undefined property: stdClass::$counterPK
in APPPATH/Models/Telegram.php on line 122.
 1 APPPATH/Models/Telegram.php(122): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined property: stdClass::$counterPK', 'APPPATH/Models/Telegram.php', 122, [...])
 2 APPPATH/Models/Telegram.php(13): App\Models\Telegram->callback(Object(stdClass), Object(stdClass), 971483502)
 3 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(Object(stdClass))
 4 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 5 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
 6 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-08 07:56:39 --> Trying to get property 'callback_query' of non-object
in APPPATH/Models/Telegram.php on line 11.
 1 APPPATH/Models/Telegram.php(11): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get property \'callback_query\' of non-object', 'APPPATH/Models/Telegram.php', 11, [...])
 2 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(null)
 3 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 4 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
 5 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-08 07:59:43 --> Trying to get property 'callback_query' of non-object
in APPPATH/Models/Telegram.php on line 11.
 1 APPPATH/Models/Telegram.php(11): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get property \'callback_query\' of non-object', 'APPPATH/Models/Telegram.php', 11, [...])
 2 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(null)
 3 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 4 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
 5 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-08 08:15:40 --> Trying to get property 'callback_query' of non-object
in APPPATH/Models/Telegram.php on line 11.
 1 APPPATH/Models/Telegram.php(11): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get property \'callback_query\' of non-object', 'APPPATH/Models/Telegram.php', 11, [...])
 2 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(null)
 3 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 4 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
 5 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-08 08:17:08 --> Trying to get property 'callback_query' of non-object
in APPPATH/Models/Telegram.php on line 11.
 1 APPPATH/Models/Telegram.php(11): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get property \'callback_query\' of non-object', 'APPPATH/Models/Telegram.php', 11, [...])
 2 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(null)
 3 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 4 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
 5 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-08 08:19:39 --> Trying to get property 'callback_query' of non-object
in APPPATH/Models/Telegram.php on line 11.
 1 APPPATH/Models/Telegram.php(11): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get property \'callback_query\' of non-object', 'APPPATH/Models/Telegram.php', 11, [...])
 2 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(null)
 3 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 4 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
 5 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-08 08:21:02 --> Trying to get property 'callback_query' of non-object
in APPPATH/Models/Telegram.php on line 11.
 1 APPPATH/Models/Telegram.php(11): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get property \'callback_query\' of non-object', 'APPPATH/Models/Telegram.php', 11, [...])
 2 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(null)
 3 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 4 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
 5 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-08 08:21:05 --> Trying to get property 'callback_query' of non-object
in APPPATH/Models/Telegram.php on line 11.
 1 APPPATH/Models/Telegram.php(11): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get property \'callback_query\' of non-object', 'APPPATH/Models/Telegram.php', 11, [...])
 2 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(null)
 3 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 4 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
 5 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-08 08:21:06 --> Trying to get property 'callback_query' of non-object
in APPPATH/Models/Telegram.php on line 11.
 1 APPPATH/Models/Telegram.php(11): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get property \'callback_query\' of non-object', 'APPPATH/Models/Telegram.php', 11, [...])
 2 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(null)
 3 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 4 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
 5 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-08 08:36:13 --> Trying to get property 'callback_query' of non-object
in APPPATH/Models/Telegram.php on line 11.
 1 APPPATH/Models/Telegram.php(11): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get property \'callback_query\' of non-object', 'APPPATH/Models/Telegram.php', 11, [...])
 2 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(null)
 3 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 4 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
 5 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-08 09:03:25 --> Trying to get property 'counterPk' of non-object
in APPPATH/Models/Telegram.php on line 46.
 1 APPPATH/Models/Telegram.php(46): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get property \'counterPk\' of non-object', 'APPPATH/Models/Telegram.php', 46, [...])
 2 APPPATH/Models/Telegram.php(13): App\Models\Telegram->message(Object(stdClass), Object(stdClass), 971483502)
 3 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(Object(stdClass))
 4 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 5 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
 6 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-08 09:14:02 --> Trying to get property 'callback_query' of non-object
in APPPATH/Models/Telegram.php on line 11.
 1 APPPATH/Models/Telegram.php(11): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get property \'callback_query\' of non-object', 'APPPATH/Models/Telegram.php', 11, [...])
 2 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(null)
 3 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 4 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
 5 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-08 09:14:11 --> Trying to get property 'callback_query' of non-object
in APPPATH/Models/Telegram.php on line 11.
 1 APPPATH/Models/Telegram.php(11): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get property \'callback_query\' of non-object', 'APPPATH/Models/Telegram.php', 11, [...])
 2 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(null)
 3 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 4 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
 5 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-08 09:15:38 --> Trying to get property 'callback_query' of non-object
in APPPATH/Models/Telegram.php on line 11.
 1 APPPATH/Models/Telegram.php(11): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get property \'callback_query\' of non-object', 'APPPATH/Models/Telegram.php', 11, [...])
 2 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(null)
 3 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 4 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
 5 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-08 09:18:11 --> Trying to get property 'callback_query' of non-object
in APPPATH/Models/Telegram.php on line 11.
 1 APPPATH/Models/Telegram.php(11): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get property \'callback_query\' of non-object', 'APPPATH/Models/Telegram.php', 11, [...])
 2 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(null)
 3 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 4 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
 5 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-08 09:18:11 --> Trying to get property 'callback_query' of non-object
in APPPATH/Models/Telegram.php on line 11.
 1 APPPATH/Models/Telegram.php(11): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get property \'callback_query\' of non-object', 'APPPATH/Models/Telegram.php', 11, [...])
 2 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(null)
 3 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 4 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
 5 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-08 09:18:11 --> Trying to get property 'callback_query' of non-object
in APPPATH/Models/Telegram.php on line 11.
 1 APPPATH/Models/Telegram.php(11): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get property \'callback_query\' of non-object', 'APPPATH/Models/Telegram.php', 11, [...])
 2 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(null)
 3 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 4 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
 5 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-08 09:18:52 --> Trying to get property 'callback_query' of non-object
in APPPATH/Models/Telegram.php on line 11.
 1 APPPATH/Models/Telegram.php(11): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get property \'callback_query\' of non-object', 'APPPATH/Models/Telegram.php', 11, [...])
 2 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(null)
 3 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 4 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
 5 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-08 09:19:01 --> Trying to get property 'callback_query' of non-object
in APPPATH/Models/Telegram.php on line 11.
 1 APPPATH/Models/Telegram.php(11): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get property \'callback_query\' of non-object', 'APPPATH/Models/Telegram.php', 11, [...])
 2 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(null)
 3 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 4 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
 5 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-08 09:19:02 --> Trying to get property 'callback_query' of non-object
in APPPATH/Models/Telegram.php on line 11.
 1 APPPATH/Models/Telegram.php(11): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get property \'callback_query\' of non-object', 'APPPATH/Models/Telegram.php', 11, [...])
 2 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(null)
 3 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 4 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
 5 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-08 09:20:02 --> Trying to get property 'callback_query' of non-object
in APPPATH/Models/Telegram.php on line 11.
 1 APPPATH/Models/Telegram.php(11): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get property \'callback_query\' of non-object', 'APPPATH/Models/Telegram.php', 11, [...])
 2 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(null)
 3 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 4 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
 5 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-08 09:20:04 --> Trying to get property 'callback_query' of non-object
in APPPATH/Models/Telegram.php on line 11.
 1 APPPATH/Models/Telegram.php(11): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get property \'callback_query\' of non-object', 'APPPATH/Models/Telegram.php', 11, [...])
 2 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(null)
 3 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 4 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
 5 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-08 09:20:05 --> Trying to get property 'callback_query' of non-object
in APPPATH/Models/Telegram.php on line 11.
 1 APPPATH/Models/Telegram.php(11): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get property \'callback_query\' of non-object', 'APPPATH/Models/Telegram.php', 11, [...])
 2 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(null)
 3 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 4 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
 5 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-08 09:20:07 --> Trying to get property 'callback_query' of non-object
in APPPATH/Models/Telegram.php on line 11.
 1 APPPATH/Models/Telegram.php(11): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get property \'callback_query\' of non-object', 'APPPATH/Models/Telegram.php', 11, [...])
 2 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(null)
 3 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 4 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
 5 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-08 09:20:18 --> Trying to get property 'callback_query' of non-object
in APPPATH/Models/Telegram.php on line 11.
 1 APPPATH/Models/Telegram.php(11): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get property \'callback_query\' of non-object', 'APPPATH/Models/Telegram.php', 11, [...])
 2 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(null)
 3 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 4 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
 5 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-08 09:21:14 --> Trying to get property 'callback_query' of non-object
in APPPATH/Models/Telegram.php on line 11.
 1 APPPATH/Models/Telegram.php(11): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get property \'callback_query\' of non-object', 'APPPATH/Models/Telegram.php', 11, [...])
 2 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(null)
 3 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 4 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
 5 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-08 09:21:16 --> Trying to get property 'callback_query' of non-object
in APPPATH/Models/Telegram.php on line 11.
 1 APPPATH/Models/Telegram.php(11): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get property \'callback_query\' of non-object', 'APPPATH/Models/Telegram.php', 11, [...])
 2 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(null)
 3 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 4 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
 5 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-08 09:22:13 --> Trying to get property 'callback_query' of non-object
in APPPATH/Models/Telegram.php on line 11.
 1 APPPATH/Models/Telegram.php(11): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get property \'callback_query\' of non-object', 'APPPATH/Models/Telegram.php', 11, [...])
 2 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(null)
 3 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 4 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
 5 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-08 09:23:32 --> Trying to get property 'callback_query' of non-object
in APPPATH/Models/Telegram.php on line 11.
 1 APPPATH/Models/Telegram.php(11): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get property \'callback_query\' of non-object', 'APPPATH/Models/Telegram.php', 11, [...])
 2 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(null)
 3 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 4 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
 5 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-08 09:23:36 --> Trying to get property 'callback_query' of non-object
in APPPATH/Models/Telegram.php on line 11.
 1 APPPATH/Models/Telegram.php(11): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get property \'callback_query\' of non-object', 'APPPATH/Models/Telegram.php', 11, [...])
 2 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(null)
 3 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 4 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
 5 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-08 09:24:07 --> Trying to get property 'callback_query' of non-object
in APPPATH/Models/Telegram.php on line 11.
 1 APPPATH/Models/Telegram.php(11): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get property \'callback_query\' of non-object', 'APPPATH/Models/Telegram.php', 11, [...])
 2 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(null)
 3 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 4 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
 5 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-08 09:25:38 --> Trying to get property 'callback_query' of non-object
in APPPATH/Models/Telegram.php on line 11.
 1 APPPATH/Models/Telegram.php(11): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get property \'callback_query\' of non-object', 'APPPATH/Models/Telegram.php', 11, [...])
 2 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(null)
 3 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 4 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
 5 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-08 09:27:40 --> Trying to get property 'callback_query' of non-object
in APPPATH/Models/Telegram.php on line 11.
 1 APPPATH/Models/Telegram.php(11): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get property \'callback_query\' of non-object', 'APPPATH/Models/Telegram.php', 11, [...])
 2 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(null)
 3 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 4 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
 5 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-08 09:27:53 --> Trying to get property 'callback_query' of non-object
in APPPATH/Models/Telegram.php on line 11.
 1 APPPATH/Models/Telegram.php(11): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get property \'callback_query\' of non-object', 'APPPATH/Models/Telegram.php', 11, [...])
 2 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(null)
 3 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 4 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
 5 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-08 09:33:11 --> Trying to get property 'callback_query' of non-object
in APPPATH/Models/Telegram.php on line 11.
 1 APPPATH/Models/Telegram.php(11): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get property \'callback_query\' of non-object', 'APPPATH/Models/Telegram.php', 11, [...])
 2 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(null)
 3 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 4 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
 5 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-08 09:38:01 --> Trying to get property 'callback_query' of non-object
in APPPATH/Models/Telegram.php on line 11.
 1 APPPATH/Models/Telegram.php(11): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get property \'callback_query\' of non-object', 'APPPATH/Models/Telegram.php', 11, [...])
 2 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(null)
 3 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 4 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
 5 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-08 09:39:01 --> Trying to get property 'callback_query' of non-object
in APPPATH/Models/Telegram.php on line 11.
 1 APPPATH/Models/Telegram.php(11): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get property \'callback_query\' of non-object', 'APPPATH/Models/Telegram.php', 11, [...])
 2 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(null)
 3 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 4 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
 5 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-08 09:39:21 --> Trying to get property 'callback_query' of non-object
in APPPATH/Models/Telegram.php on line 11.
 1 APPPATH/Models/Telegram.php(11): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get property \'callback_query\' of non-object', 'APPPATH/Models/Telegram.php', 11, [...])
 2 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(null)
 3 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 4 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
 5 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-08 09:40:08 --> Trying to get property 'callback_query' of non-object
in APPPATH/Models/Telegram.php on line 11.
 1 APPPATH/Models/Telegram.php(11): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get property \'callback_query\' of non-object', 'APPPATH/Models/Telegram.php', 11, [...])
 2 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(null)
 3 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 4 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
 5 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-08 09:40:50 --> Trying to get property 'callback_query' of non-object
in APPPATH/Models/Telegram.php on line 11.
 1 APPPATH/Models/Telegram.php(11): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get property \'callback_query\' of non-object', 'APPPATH/Models/Telegram.php', 11, [...])
 2 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(null)
 3 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 4 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
 5 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-08 09:42:37 --> 7 : Failed to connect to 10.10.5.173 port 8081: Connection refused
in SYSTEMPATH/HTTP/CURLRequest.php on line 655.
 1 SYSTEMPATH/HTTP/CURLRequest.php(655): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('7', 'Failed to connect to 10.10.5.173 port 8081: Connection refused')
 2 SYSTEMPATH/HTTP/CURLRequest.php(363): CodeIgniter\HTTP\CURLRequest->sendRequest([...])
 3 SYSTEMPATH/HTTP/CURLRequest.php(136): CodeIgniter\HTTP\CURLRequest->send('post', 'http://10.10.5.173:8081/sendMessage/')
 4 SYSTEMPATH/HTTP/CURLRequest.php(209): CodeIgniter\HTTP\CURLRequest->request('post', 'http://10.10.5.173:8081/sendMessage/', [...])
 5 APPPATH/Models/HTTP.php(13): CodeIgniter\HTTP\CURLRequest->post('http://10.10.5.173:8081/sendMessage/', [...])
 6 APPPATH/Models/Telegram.php(161): App\Models\HTTP->request([...])
 7 APPPATH/Models/Telegram.php(24): App\Models\Telegram->sendMessage(971483502, [...])
 8 APPPATH/Models/Telegram.php(13): App\Models\Telegram->message(null, Object(stdClass), 971483502)
 9 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(Object(stdClass))
10 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
11 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
12 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-08 09:42:37 --> Uncaught CodeIgniter\Format\Exceptions\FormatException: Failed to parse json string, error: "Type is not supported". in /ext/WWW/konex.com.ua/zayavki/system/Format/JSONFormatter.php:41
Stack trace:
#0 /ext/WWW/konex.com.ua/zayavki/system/Format/JSONFormatter.php(41): CodeIgniter\Format\Exceptions\FormatException::forInvalidJSON('Type is not sup...')
#1 /ext/WWW/konex.com.ua/zayavki/system/API/ResponseTrait.php(348): CodeIgniter\Format\JSONFormatter->format(Array)
#2 /ext/WWW/konex.com.ua/zayavki/system/API/ResponseTrait.php(99): CodeIgniter\Debug\Exceptions->format(Array)
#3 /ext/WWW/konex.com.ua/zayavki/system/Debug/Exceptions.php(133): CodeIgniter\Debug\Exceptions->respond(Array, 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(CodeIgniter\HTTP\Exceptions\HTTPException))
#5 {main}
  thrown
in SYSTEMPATH/Format/JSONFormatter.php on line 41.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2023-03-08 09:43:39 --> Trying to get property 'counterPk' of non-object
in APPPATH/Models/Telegram.php on line 46.
 1 APPPATH/Models/Telegram.php(46): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get property \'counterPk\' of non-object', 'APPPATH/Models/Telegram.php', 46, [...])
 2 APPPATH/Models/Telegram.php(13): App\Models\Telegram->message(Object(stdClass), Object(stdClass), 971483502)
 3 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(Object(stdClass))
 4 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 5 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
 6 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-08 09:43:40 --> Trying to get property 'counterPk' of non-object
in APPPATH/Models/Telegram.php on line 46.
 1 APPPATH/Models/Telegram.php(46): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get property \'counterPk\' of non-object', 'APPPATH/Models/Telegram.php', 46, [...])
 2 APPPATH/Models/Telegram.php(13): App\Models\Telegram->message(Object(stdClass), Object(stdClass), 971483502)
 3 APPPATH/Controllers/Meters.php(158): App\Models\Telegram->telegram(Object(stdClass))
 4 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Meters->telegram()
 5 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meters))
 6 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
